docassemble/AssemblyLine/sessions.py

```diff
@@ -965,10 +1147,18 @@ def load_interview_json(
     variables_to_filter: Optional[List[str]] = None,
 ) -> Optional[int]:
     """
-    Provided a JSON string, load the specified variables into a Docassemble session. JSON with annotated class names
-    will be processed into Docassemble objects.
-
-    If new_session is not provided, the JSON answers will be loaded into the current interview.
+    Given a JSON string, this function loads the specified variables into a Docassemble session.
+    JSON strings containing annotated class names will be transformed into Docassemble objects.
+    If the `new_session` argument is not set, the JSON answers will be loaded into the current interview.
+
+    Args:
+        json_string (str): A JSON-formatted string containing session variables.
+        new_session (bool, optional): Specifies whether to create a new session or load into the current one. Defaults to False.
+        new_interview_filename (Optional[str], optional): Filename for the new session. Defaults to None.
+        variables_to_filter (Optional[List[str]], optional): List of variables to exclude. Defaults to None.
+
+    Returns:
+        Optional[int]: ID of the newly created session if `new_session` is True, otherwise True or False based on success.
```

Comment:
```suggestion
        Optional[Union[int, bool]]: ID of the newly created session if `new_session` is True, otherwise True or False based on success.
```